src/purifier.js

```diff
@@ -72,11 +72,12 @@ let KEY_COUNTER = 0;
  * Returns a <body> element containing the sanitized, serialized `dirty`.
  * @param {string} dirty
  * @param {boolean=} diffing
+ * @param {Document=} opt_doc
  * @return {!Node}
  */
-export function purifyHtml(dirty, diffing = false) {
+export function purifyHtml(dirty, diffing = false, opt_doc) {
```

Comment:
Let's require the `Document` param. It'll avoid bugs where the caller forgets to pass it in.

```js
export function purifyHtml(doc, dirty, diffing = false) {
```